app/templates/layout.html

```diff
@@ -78,5 +78,6 @@
 
     <script src="{{ basepath }}/components/jquery/jquery.js"></script>
     <script src="{{ basepath }}/components/bootstrap/bootstrap-built.js"></script>
+    <script src="{{ basepath }}/js/equalHeightColumns.js"></script>
```

Comment:
Sure. I'm completely ignorant of how this framework works though. How would you suggest it be added instead? Placing it just above the content endblock doesn't work because it depends on jquery. Would I need to add a block to layout.html specifically for adding additional scripts? 

I'd also like to rename the javascript file to something more specific. equalHeightBoxes for example?